client/src/actions/gameAction.js

```diff
@@ -16,14 +17,15 @@ export const updateCurrentQuestion = (currentQues) => async (dispatch) => {
   });
 };
 
-export const updateScore = (score) => async (dispatch) => {
-  dispatch({
-    type: SCORE,
-    score
-  });
-};
-
-export const updateGameResult = (score) => async (dispatch) => {
+export const updateScore = (score) => async (dispatch, getState) => {
+  const state = getState();
+  let currentGame = state.contractStatus.currentGame;
+  var scoreEncrypt = CryptoJS.AES.encrypt(
+    currentGame.roomId + '+' + score.toString(),
```

Comment:
ko nên dùng dấu `+` sẽ dễ gây nhầm lẫn hoặc biết đâu bị eval ở hàm nào đó, vì bản thân cộng string cũng là một phép toán
nên dùng underscore `_` thay thế thì hợp lý hơn